test/parallel/test-whatwg-readablebytestream.js

```diff
@@ -209,9 +212,11 @@ class Source {
   reader.releaseLock();
   assert.rejects(reader.read(new Uint8Array(10)), {
     code: 'ERR_INVALID_STATE',
+    message: 'Invalid state: The reader is not attached to a stream',
```

Comment:
We don't test error message, see https://github.com/nodejs/node/pull/42813
```suggestion
```

Reply:
@aduh95 Thanks, I change that checking only the `code` property.